wkcuber/api/properties/layer_properties.py

```diff
@@ -46,10 +46,10 @@ def _extract_num_channels(
 
 
 properties_floating_type_to_python_type = {
-    "float": np.dtype("float32"),
-    np.float: np.dtype("float32"),
-    float: np.dtype("float32"),
-    "double": np.dtype("float64"),
+    "float": "float32",
+    np.float: "float32",
+    float: "float32",
```

Comment:
Not sure if those are actually needed, since they can't be part of the json, can they? Doesn't matter too much, though.